core/app/modules/import-export/runners/revert/templates.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Elementor\Core\App\Modules\ImportExport\Runners\Revert;
+
+use Elementor\Core\App\Modules\ImportExport\Utils as ImportExportUtils;
+use Elementor\Core\Base\Document;
+use Elementor\Core\Documents_Manager;
+use Elementor\Plugin;
+use Elementor\TemplateLibrary\Source_Local;
+use Elementor\Utils;
+
+class Templates extends Revert_Runner_Base {
```

Comment:
Add a comment that the actual implementation is part of the pro plugin

Reply:
Done